ic-asset/src/asset_config.rs

```diff
@@ -27,6 +27,7 @@ struct AssetConfigRule {
     r#match: GlobMatcher,
     cache: Option<CacheConfig>,
     headers: Maybe<HeadersConfig>,
+    ignore_hidden: bool,
```

Comment:
I'd suggest to make this about "ignoring files", rather than about "ignoring hidden files."

Where the default heuristic is:
- filename starts with a dot: ignore the file
- filename does not start with a dot: include the file

The `ignore` field, if present, overrides the above behavior (for any file)

```suggestion
    ignore: Option<bool>,
```

Reply:
While I think this suggestion is a little bit outside of what this PR is trying to achieve (the PR was conceived due to the need to include `.well-known/*` assets in asset upload), I think it's overall a good idea, and makes everything "endgame-customizable".

I'll include this in this PR, because I don't want our users to get confused about the change 
> `ignore_hidden` is not `ignore`